account-kit/react/src/tailwind/components/buttons.ts

```diff
@@ -1,7 +1,17 @@
 import type { ComponentDef } from "../types";
 
 export const buttonComponents: ComponentDef = {
+  ".btn-group": {
```

Comment:
For my own understanding, how are these CSS rules localized to the account-kit UI component? I'm assuming this isn't changing global stylings for common classes like `.btn`, is it?

Reply:
I need to double check, I think they might conflict with global styles actually. I need to double check, but I think we might need to configure our plugin to use a prefix to avoid the clashing

Reply:
this probably is a problem, but will have to deal with it in a separate PR